src/initializers/koa/add-request-id.ts

```diff
@@ -5,7 +5,7 @@ export default function (config): Middleware {
   return async function addRequestId(ctx: Context, next: () => void) {
     let requestId = ctx.headers[config.traceHeaderName.toLowerCase()];
     if (!requestId) {
-      requestId = `orka-${randomUUID()}`;
+      requestId = `orka:${randomUUID()}`;
```

Comment:
❓ Why you changed that?

Reply:
Personal preference. It was introduced in previous pr not currently in any orka version.